modin/core/execution/ray/common/engine_wrapper.py

```diff
@@ -307,6 +307,31 @@ def post_materialize(self, materialized):
         """
         raise NotImplementedError()
 
+    def __reduce__(self):
+        """
+        Replace this hook with the materialized object on serialization.
+
+        Returns
+        -------
+        tuple
+        """
+        return _hook_deserializer, (RayWrapper.materialize(self),)
+
+
+def _hook_deserializer(obj):
```

Comment:
Let's make this function a staticmethod or a classmethod. This way we can move it closer to the place of use.

Reply:
When moved inside the class, the serialized data requires 20 bytes more, because the class name is also serialized.

Reply:
Moved, renamed to _get() and added check - if the object is int, this type is used instead because it has a 3x smaller serialized form.